practice-one/pages/index.html

```diff
@@ -0,0 +1,605 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="../styles/main.css">
+  <link rel="stylesheet" href="../assets/font-awesome/css/all.css">
+</head>
+<body>
+  <!-- Start Header -->
+  <header class="header section">
+    <div class="header-container container">
+      <div class="header-nav">
+        <a href="#" class="logo"><span>Digi</span>ency</a>
+        <nav>
+          <ul class="nav">
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">Home</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">About Us</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">Services</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">Portfolio</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">Blog</a>
+            </li>
+            <li class="nav-item">
+              <a href="#" class="nav-item-link">Contact Us</a>
+            </li>
+          </ul>
+          <button class="btn">Let's Talk</button>
+        </nav>
+      </div>
+
+      <div class="header-content">
+        <div class="header-content-text">
+          <h1 class="title-biggest">Choose What Matters To Your Business & Your Customers.</h1>
+          <p class="title-normal">Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus, perferendis rerum. 
```

Comment:
this is not title, it is a paragraph, then class should be something `description`

Reply:
Thank you for your comments. I will update it